modulemd/tests/test-modulemd-packager-v3.c

```diff
@@ -402,192 +387,44 @@ packager_test_map_to_stream_v2_autoname (void)
-  v3_stream = modulemd_packager_v3_to_stream_v3 (packager, &error);
-  g_assert_no_error (error);
-  g_assert_nonnull (v3_stream);
-  g_assert_true (MODULEMD_IS_MODULE_STREAM_V3 (v3_stream));
-
-  /* confirm everything comes back that we expect */
-  g_assert_null (modulemd_module_stream_get_module_name (
-    MODULEMD_MODULE_STREAM (v3_stream)));
-  g_assert_null (modulemd_module_stream_get_stream_name (
-    MODULEMD_MODULE_STREAM (v3_stream)));
-
-  g_assert_cmpstr (
-    "Summary", ==, modulemd_module_stream_v3_get_summary (v3_stream, "C"));
-  g_assert_cmpstr ("Description",
-                   ==,
-                   modulemd_module_stream_v3_get_description (v3_stream, "C"));
-  g_assert_cmpstr (
-    "f33", ==, modulemd_module_stream_v3_get_platform (v3_stream));
-
-  /* a module license is not required for StreamV3 as it is for StreamV2, so
-   * the license list should be an empty
-   */
-  list = modulemd_module_stream_v3_get_module_licenses_as_strv (v3_stream);
-  g_assert_nonnull (list);
-  g_assert_null (list[0]);
-  g_clear_pointer (&list, g_strfreev);
-
-  /* PackagerV3 to Index conversion should automatically generate a module and
-   * stream name so it can be added to an index
-   */
-  index = modulemd_packager_v3_to_stream_v3_ext (packager, &error);
-  g_assert_no_error (error);
-  g_assert_nonnull (index);
-  g_assert_true (MODULEMD_IS_MODULE_INDEX (index));
-
-  /* however, the automatic module/stream names should not appear in an index dump */
-  yaml_str = modulemd_module_index_dump_to_string (index, &error);
-  g_assert_no_error (error);
-  g_assert_nonnull (yaml_str);
-  g_assert_cmpstr (yaml_str,
-                   ==,
-                   "---\n"
-                   "document: modulemd-stream\n"
-                   "version: 3\n"
-                   "data:\n"
-                   "  context: ctx42\n"
-                   "  summary: Summary\n"
-                   "  description: >-\n"
-                   "    Description\n"
-                   "  dependencies:\n"
-                   "    platform: f33\n"
-                   "...\n");
-
-  g_clear_pointer (&yaml_str, g_free);
-  g_clear_object (&v3_stream);
-  g_clear_object (&packager);
-  g_clear_object (&index);
-
-  /* restore default mdversion to avoid unexpected results from other tests */
-  modulemd_set_default_stream_mdversion (default_mdv);
-}
 
 static void
 packager_test_read_to_index (void)
 {
   gboolean ret;
+  guint i;
   g_autoptr (ModulemdModuleIndex) index = NULL;
+  g_autoptr (ModulemdSubdocumentInfo) subdoc = NULL;
   g_autoptr (GError) error = NULL;
   g_autoptr (GPtrArray) failures = NULL;
   g_autofree gchar *yaml_path = NULL;
   g_autofree gchar *yaml_str = NULL;
   g_autofree gchar *expected_path = NULL;
   g_autofree gchar *expected_str = NULL;
-  ModulemdModuleStreamVersionEnum default_mdv;
-
-  /* get and save current default stream mdversion */
-  default_mdv = modulemd_get_default_stream_mdversion ();
 
   /* create a stream v2 index */
```

Comment:
```suggestion
  /* create an index */
```
Or just drop this comment.